src/explore-education-statistics-common/src/modules/data-catalogue/components/ChangeList.tsx

```diff
@@ -0,0 +1,94 @@
+import ChangeItem from '@common/modules/data-catalogue/components/ChangeItem';
+import groupByChangeType from '@common/modules/data-catalogue/utils/groupByChangeType';
+import { Change, ChangeSet } from '@common/services/types/apiDataSetChanges';
+import React from 'react';
+
+const metaTypeLabels = {
+  filters: 'filters',
+  filterOptions: 'filter options',
+  geographicLevels: 'geographic level options',
+  indicators: 'indicators',
+  locationGroups: 'location groups',
+  locationOptions: 'location options',
+  timePeriods: 'time periods',
+} satisfies Record<keyof ChangeSet, string>;
+
+export interface ChangeListProps<TState> {
+  changes?: Change<TState>[];
+  metaType: keyof ChangeSet;
+  metaTypeLabel?: string;
+  testId?: string;
+}
+
+export default function ChangeList<TState>({
+  changes = [],
+  metaType,
+  metaTypeLabel: customMetaTypeLabel,
+  testId,
+}: ChangeListProps<TState>) {
+  if (!changes.length) {
+    return null;
+  }
+
+  const { additions, deletions, updates } = groupByChangeType(changes);
+
+  const metaTypeLabel = customMetaTypeLabel ?? metaTypeLabels[metaType];
+
+  return (
```

Comment:
This might be a easier to visually parse if was grouped by the meta type, it currently feels a bit unordered if you end up with something like this:
![Screenshot 2024-08-12 131549](https://github.com/user-attachments/assets/b6478f73-3077-4c33-82ae-a43f11aa5148)

Something like this might work better:

![Screenshot 2024-08-12 132036](https://github.com/user-attachments/assets/3af73cd7-1359-4c22-af0c-6c1e139082ac)